manifests/env/phobos/kustomization.yaml

```diff
@@ -70,3 +71,5 @@ images:
   - name: paddock
     newName: image-registry.openshift-image-registry.svc:5000/b4mad-racing/paddock
     newTag: latest
+  - name: telegraf
+    newTag: "1.27.3"
```

Comment:
shouldnt we keep it at the previous version until we know that it works on nostromo?

Reply:
exactly why I started deploying nostromo-stage